debian/changelog

```diff
@@ -1,50 +1,58 @@
 ubuntu-advantage-tools (19.5) UNRELEASED; urgency=medium
 
   * New upstream release (LP: #1832757):
-    - cc: add a warning about download size before install
-    - repo: un-comment ESM sources.list lines on repo disable
-    - attach: two-factor auth reprompt only two-factor auth on failed 2fa
-    - keyrings: regenerate keyrings on a trusty host
-    - refresh keyrings to match current production for
-      fips, fips-updates, cc-eal
-    - apt: write commented -updates lines instead of omitting them
-    - environ: allow lowercase ua_<config_option> overrides
-    - logs: no longer redact secrets as logfile is root read-only
+    - packaging:
+      + d/control: depend on libapt-pkg<ABI_VERSION> to use pin-priority never
+      + d/postinst: adjust logfile permissions
+      + d/postinst: remove public files and generate status cache on upgrade
+      + d/postinst: Remove the old CACHE_DIR in postinst
+      + d/postrm: remove log files on package purge
+      + d/postrm: remove the ESM pinning file on purge
+      + trusty should remove v1 esm key if present after upgrade
+      + keyrings: regenerate keyrings on a trusty host
+      + refresh keyrings to match current production for fips and cc-eal
+    - apt:
+      + all repo entitlements now call apt-get update on enable
+      + enable -updates if -updates from the Ubuntu archive is enabled
+      + Add basic i18n (good enough for lang packs)
+      + retry apt install and update commands 3 times simple backoff
+      + write commented -updates lines instead of omitting them
+    - attach/detach:
+      + added --no-auto-enable option
+      + suppress messages from inapplicable default entitlements
+      + two-factor auth reprompt only two-factor auth on failed 2fa
+      + honour enableByDefault obligations from contract server
+      + livepatch: no auto-enable on attach for trusty
+      + don't attempt to disable inapplicable entitlements during detach
+      + check for root before checking for attach in assert_attached_root
+    - add subcommand to refresh all contract details
```

Comment:
Let's group all lone entries like this one (that have no "+" sub entries) together somewhere (at the top or at the bottom)